modules/services/trayer.nix

```
@@ -0,0 +1,85 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.trayer;
+
+in {
+  meta.maintainers = [ maintainers.mager ];
+
+  options = {
+    services.trayer = {
+      enable =
+        mkEnableOption "trayer the lightweight GTK2+ systray for UNIX desktops";
```

nit. As is, this will be rendered as:
> Whether or not to enable trayer the lightweight GTK2+ systray for UNIX desktops.
in the generated docs.

```suggestion
        mkEnableOption "trayer, the lightweight GTK2+ systray for UNIX desktops";
```